packages/@ember/-internals/runtime/lib/mixins/-proxy.js

```diff
@@ -58,7 +59,16 @@ export default Mixin.create({
   }),
 
   [UNKNOWN_PROPERTY_TAG](key) {
-    return combine(getChainTagsForKey(this, `content.${key}`));
+    let tag = tagFor(this, key);
+
+    if (DEBUG) {
+      setupMandatorySetter(tag, this, key);
```

Comment:
Doesn't this mean that we automatically define each of these keys on the proxy instance? E.g. `Object.keys` output would be changed.

That seems a bit wrong, no?

Reply:
It does, my mistake there. We should likely have a test for this.